This warning causes errors when compiling tests in C++ code generated by Cython,
In particular: external/com_github_grpc_grpc/src/python/grpcio/grpc/_cython/cygrpc.cpp
see https://github.com/cython/cython/issues/7934

Newer GRPC versions don't run into this, but we don't need this warning at all.

Author: Alexander Grund (TU Dresden)

diff --git a/.bazelrc b/.bazelrc
--- a/.bazelrc
+++ b/.bazelrc
@@ -423,9 +423,6 @@ common:linux --copt="-Wno-deprecated-declarations"
 common:linux --copt="-Wno-ignored-attributes"
 common:linux --copt="-Wno-array-bounds"
 
-# Add unused-result as an error on Linux.
-common:linux --copt="-Wunused-result"
-common:linux --copt="-Werror=unused-result"
 # Add switch as an error on Linux.
 common:linux --copt="-Wswitch"
 common:linux --copt="-Werror=switch"
